playwright.config.js

```diff
@@ -38,15 +38,15 @@ module.exports = defineConfig({
             use: { ...devices['Desktop Chrome'] },
         },
 
-        {
-            name: 'firefox',
-            use: { ...devices['Desktop Firefox'] },
-        },
+        // // // // {
+        // // // //     name: 'firefox',
+        // // // //     use: { ...devices['Desktop Firefox'] },
+        // // // // },
```

Comment:
Why these lines are so hardly commented? You could just remove in case you dont need it. Or comment out as usual